code/process_sequences.sh

```diff
@@ -20,10 +20,14 @@ then
   echo "To run this script, supply a fasta.gz file of sequences."
   exit 1
 fi
-if [ $# -gt 1 ]
+
+# Check for ALL
+if [ "$#" -eq 2 ]
 then
-  echo "Too many parameters passed. Supply 1 fasta.gz file."
-  exit 1
+  if [ $2 == 'ALL' ]
+  then
+    echo "ALL file detected."
```

Comment:
@naupaka @mhaliz @melt6457 Thanks for the detailed feedback. I've added logic, but I'm still unsure of what content goes inside the nested if statements in order to make all the previous output display properly(?).

Reply:
ALL is not a file. If there is ALL, then do all the things. If there is not ALL, then only do the one thing.

Reply:
Sounds kind of like a Zen koan.